accelerators/frontend/micro-ui/web/micro-ui-internals/packages/react-components/src/hoc/FormComposerV2.js

```diff
@@ -417,6 +417,7 @@ export const FormComposer = (props) => {
               />
             )}
             name={config.key}
+            rules={!disableFormValidation ? { required: isMandatory, ...populators.validation } : {}}
```

Comment:
can you check the components ie project create, estimate create screens it doesn't made any impact
@Tulika-eGov